index.html

```diff
@@ -50,19 +50,19 @@
                 <li class="navbar-item" role="menuitem">
                   <button
                     class="button is-white is-outlined with-sublist" aria-controls="services-menu"
-                    aria-haspopup="menu" aria-expanded="false">
+                    aria-haspopup="menu" aria-expanded="false" data-target="services-menu" >
                     <span class="icon" aria-hidden="true">
                       <i class="fab fa-superpowers"></i>
                     </span>
                     <span>Services</span>
                   </button>
-                  <!-- TODO: Remove hidden class and aria-hidden -->
                   <ul class="sublist hidden" id="services-menu" role="menu" aria-hidden="true">
-                    <li role="menuitem">Design</li>
-                    <li role="menuitem">Consult</li>
-                    <li role="menuitem">Engineer</li>
-                    <li role="menuitem">Operate</li>
-                    <li role="menuitem">Optimize</li>
+                    <li role="menuitem" tabindex="0">Design</li>
+                    <li role="menuitem" tabindex="0">Consult</li>
+                    <li role="menuitem" tabindex="0">Engineer</li>
+                    <li role="menuitem" tabindex="0">Operate</li>
+                    <hr role="separator" class="separator">
```

Comment:
Looks like separator needs to be added to main menu, but I decided that in submenu is faster.

Reply:
No, you did it correct. Please check the example https://www.w3.org/WAI/ARIA/apg/patterns/menubar/examples/menubar-navigation/ (Admissions menu item)